.github/workflows/setup.sh

```diff
@@ -18,14 +18,6 @@ else
    spack load intel-oneapi-mkl
 fi
 
-if [ "$COMPILER" = "intel" ]; then
-   spack load intel-oneapi-compilers
```

Comment:
Still need to load the intel compilers

Reply:
Thanks, hopefully all checks should now pass